src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -105,6 +105,21 @@ public void BuildBasicApplication ([Values (true, false)] bool isRelease, [Value
 			}
 		}
 
+		[Test]
+		public void BasicApplicationOtherRuntime ([Values (true, false)] bool isRelease)
+		{
+			var proj = new XamarinAndroidApplicationProject {
+				IsRelease = isRelease,
+				// Add locally downloaded CoreCLR packs
+				ExtraNuGetConfigSources = {
+					Path.Combine (XABuildPaths.BuildOutputDirectory, "nuget-unsigned"),
+				}
+			};
+			proj.SetProperty ("UseMonoRuntime", "false"); // Enables CoreCLR
```

Comment:
Is this one bool property going to be enough? As we will have MonoVM, CoreCLR and NativeAOT?

Reply:
The logic is here:
https://github.com/dotnet/android/blob/44c5c30d3da692c54ca27d4a41571ef20b73670f/src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets#L29-L31

But yeah, I think they need to invent a new property. `$(PublishAot)` and `$(UseMonoRuntime)` are what iOS uses currently.